src/metrics/database_definitions.go

```diff
@@ -45,6 +45,7 @@ func (q *QueryDefinition) insertDatabaseNames(databases collection.DatabaseList)
 var databaseDefinitionUnder91 = &QueryDefinition{
 	query: `SELECT -- UNDER91
 		D.datname AS database,
+		(SELECT setting::INTEGER FROM pg_settings WHERE  name = 'max_connections')max_connections,
```

Comment:
This query seems a bit different from the others, might be a good idea to put a small comment somewhere indicating why it's the case (e.g. belongs to a different table).

Also I've noticed there is no `AS` token. This is probably fine but just wanted to doublecheck.

Reply:
Added comments on the line.
The `AS` is  not mandatory but added for consistency.

Thx!